src/CMakeLists.txt

```diff
@@ -236,12 +236,20 @@ ADD_FEATURE_INFO(ntlmclient GIT_NTLM "NTLM authentication support for Unix")
 # Optional external dependency: libgssapi
 IF (USE_GSSAPI)
 	FIND_PACKAGE(GSSAPI)
+	IF (APPLE)
+		INCLUDE(FindGSSFramework)
+	ENDIF()
```

Comment:
I think that if there's multiple possible backends, we should expose this as another CMake option so that one may explicitly choose the backend.

Reply:
That might make sense.  I can imagine gssapi getting installed via homebrew and you want to select one or the other.

Reply:
Isn't it always about backends these days ? 😆

I'll add the selection code, but I mostly filed that so that it could be "runtime checked" before going in, since there's interest. Not just yet though 😉.

Reply:
Yeah, it is. I wonder whether we could write CMake modules to handle these with less boilerplate and more consistently. Definitely not required to be part of this PR, though. Maybe I'll some day find some time to tackle this.